week-2/interpret/time-format.js

```diff
@@ -22,15 +22,37 @@ console.log(formatTimeDisplay(61));
 // Questions
 
 // a) Run this program with node, identify the line the error is thrown. Explain why this error is occurring. How can you fix this error?
+//TypeError: num.toString(...).padStart is not a function
```

Comment:
This `TypeError` is actually due to an issue with Python Tutor: Python Tutor doesn't recognise `padStart` as a function. Can you work out why? Try asking someone on Slack / in class on Saturday...

Reply:
okay I will ask my fellow trainees on Saturday why Python Tutor does not recognise padStart as a function